dependent/acta-neuropathologica-communications.csl

```diff
@@ -6,9 +6,9 @@
     <title-short>Acta Neuropathol Commun</title-short>
     <id>http://www.zotero.org/styles/acta-neuropathologica-communications</id>
     <link href="http://www.zotero.org/styles/acta-neuropathologica-communications" rel="self"/>
-    <link href="http://www.zotero.org/styles/springer-basic-brackets" rel="independent-parent"/>
+    <link href="http://www.zotero.org/styles/springer-basic-author-date" rel="independent-parent"/>
     <link href="http://www.actaneurocomms.org" rel="documentation"/>
-    <category citation-format="numeric"/>
+    <category citation-format="author-date"/>
```

Comment:
This seems to be correct currently and the replacement looks wrong here, see http://www.actaneurocomms.org/authors/instructions/research#formatting-references and e.g. http://www.actaneurocomms.org/content/3/1/32